extension/hotfix-intermediate-2018.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no console */
+/* global ExtensionAPI */
+
+// This pref is being introduced by Bug 1954934.
+const PREF_LAST_SIGNATURE_CHECKPOINT = "extensions.signatureCheckpoint";
+
+this.hotfixIntermediateCert = class extends ExtensionAPI {
+  onStartup() {
+    if (Services.prefs.prefHasUserValue(PREF_LAST_SIGNATURE_CHECKPOINT)) {
```

Comment:
Check for `Services.appinfo.version` to exclude Firefox >= 137 so that we never need to worry about this code running in production if the user kept it around for some reason and return.